engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -39,6 +39,7 @@ class EngineJobExecutionActor(replyTo: ActorRef,
                               jobStoreActor: ActorRef,
                               callCacheReadActor: ActorRef,
                               jobTokenDispenserActor: ActorRef,
+                              backendSingletonActor: Option[ActorRef],
```

Comment:
@cjllanwarne despite what it might seem, i'm actually trying to not opine on these too much right now ;) 

@mcovarr makes a decent point in my mind, that given enough documentation perhaps this is better than what I was thinking. I was going to suggest something like `BackendManagerActor` but while the concept originally came from our convo where I was talking about an "overlord actor" that's not necessarily what this needs to be.

I think the real thing is that we need some serious (as in quality, not quantity) documentation of our backend sitch and somehow this needs to be filed under "don't just explain it in some paragraph that nobody reads"